static/js/publisher/publicise.js

```diff
@@ -53,6 +53,14 @@ const getCardPath = (snapName, options = {}) => {
     params += `summary=true`;
   }
 
+  if (options["show-screenshot"]) {
+    if (params) {
+      params = `${params}&`;
```

Comment:
This pattern is new to me, I'm not sure how I feel about it....
I'd do
```
let params = [];
params.push(param)
return `?${params.join("&")}`;
```
... I need to think about this more. I'm having a crisis.